BigBlueButton-RLP/Info.plist

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
+	<key>NSCameraUsageDescription</key>
+	<string>This app requires the access to Camera.</string>
```

Comment:
We need to come up with better descriptions to pass the app store review.
```
The camera is used to allow other participants to see the user in the web conference.
The microphone is used to allow other participants to hear the user in the web conference.
```

Reply:
Yes, agree, this is just temporary.